app/src/main/java/rectangledbmi/com/pittsburghrealtimetracker/SelectTransit.java

```diff
@@ -838,8 +841,16 @@ public void onCompleted() {
             public void onError(Throwable e) {
                 if (e.getMessage() != null && e.getLocalizedMessage() != null && !showedErrors) {
                     showedErrors = true;
-                    if (e instanceof RetrofitError) {
-                        printRetrofitError((RetrofitError) e);
+                    if (e instanceof IOException) {
+                        showToast(getString(R.string.retrofit_network_error), Toast.LENGTH_SHORT);
+                    }
+                    else if (e instanceof retrofit.HttpException) {
+                        retrofit.HttpException http = (HttpException) e;
+                        showToast(http.code() + " " + http.message() + ": "
+                                + getString(R.string.retrofit_http_error), Toast.LENGTH_SHORT);
+                    }
+                    else {
+                        showToast(e.getLocalizedMessage(), Toast.LENGTH_LONG);
```

Comment:
Instead of showing the localized error mesage in the toast, it might be better to have this in `Log.e("blah_error", e.getLocalizedMessage());` then use `getString(R.string.retrofit_unexpected_error)` in the toast.